docs/using/common.md

```diff
@@ -178,20 +185,22 @@ Example:
 
 ```bash
 # Run Jupyter Notebook classic
```

Comment:
I think this comment and the next one like this should be swapped with each other.